### PR TITLE
clean up members and add devcon.network

### PR DESCRIPTION
**Pull request by silviopaganini**

### CODE REVIEW

**Comment by silviopaganini** [comment]
knock knock @grudelsud

**Comment by grudelsud** [comment]
sorry, I'm on it.

File: src/state/reducers/members.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const initialState = {
 
 export const loadMembers = createAction('LOAD_MEMBERS', (uid) => ({
   promise: new Promise((resolve, reject) => {
+    // TODO: check it actually reloads when the collection changes
```
Comment (grudelsud) [inline src/state/reducers/members.js]: mh... feels the page doesn't reflect realtime updates. need to investigate more on this.

File: src/components/Member/MembersData.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      <Grid item xs={12} sm={12} md={4} key={member.uid}>
+        <Paper className={classes.paper}>
+          <Form
+            className="SubscriptionForm"
+            id="subscription-form"
+            getApi={this.setFormApi}
+            initialValues={member}
+            key={member.uid}
+          >
+            <MemberFields />
```
Comment (grudelsud) [inline src/components/Member/MembersData.jsx]: moved the member fields form to a separate component. didn't do the fancy testing and storybook though.

**Pull request merged**